### ISSUE TITLE
powershell module does not pass through output

### ISSUE THREAD

**Issue by mwrock** [chef/chef#4045]
If using knife via `chef.psm1` out put cannot be captured via stream redirection or variable capture. It always leaks to the console.

```
knife node list > myfile.txt
knife node list | more
$out = knife node list
```

None of these work and the first just creates an empty file.

**Comment by smurawski** [comment]
@mwrock is this only on wmf5 or have you tested other wmf versions?  I ask because WMF5 introduced a new stream (information) which parallels info out to the host.

**Comment by mwrock** [comment]
Just tested on my win 10 and not sure what Jason was testing on. I don't think this is the result of the information stream because but the way we are shelling out from the module to ruby.

**Comment by smurawski** [comment]
@mwrock I just confirmed with jason - 2012 R2 with wmf 5

**Comment by james-s-nduka** [comment]
Is there any updates to this or workarounds? I have hit a roadblock due to this issue.

**Comment by mwrock** [comment]
The workaround is not to use the module.

**Comment by btm** [comment]
This should be fixed in 12.17.49+
